mobile/lib/ui/home/home_screen.dart

```diff
@@ -28,6 +30,7 @@ class _HomeContentScreen extends StatefulWidget {
 
 class _HomeContentScreenState extends State<_HomeContentScreen> {
   final _textController = TextEditingController();
+  OptionsMenu? selectedMenu;
```

Comment:
why do you need to save this? it's just an option menu with actions when pressing the different options